flow/core/kernel/vehicle/traci.py

```diff
@@ -88,6 +93,12 @@ def __init__(self,
         # old speeds used to compute accelerations
         self.previous_speeds = {}
 
+        self.energy = Energy(self)
+        self.energy.calc_energy_level(self)
+
+        # # store speeds of all vehicles at last iteration
+        self.old_speeds = {}
+
```

Comment:
I think this was leftover from some old code, you can probably delete these lines.